.ci/append-to-github-output.sh

```diff
@@ -5,7 +5,7 @@ set -e
 NAME=$1
 VALUE=$2
 
-if [[ -z "$NAME" || -z "$VALUE" ]]; then
+if [[ $# != 2 ]]; then
```

Comment:
Why are we using string comparison operator on a number?

Reply:
https://stackoverflow.com/a/18568726/1015848

Reply:
I went with `!=` for readability. But you are right, I should put them in quotes.

Done.

Reply:
I meant to use `-ne` for integer comparison ;) but if we make everything string by `"`, that's fine too.